pkgs/games/prismlauncher/wrapper.nix

```diff
@@ -21,10 +25,28 @@
 , msaClientID ? null
 , gamemodeSupport ? stdenv.isLinux
 , textToSpeechSupport ? stdenv.isLinux
+
+# The flag `waylandSupport` enables runtime-checking of `WAYLAND_DISPLAY`;
+# if the option is enabled, a patched version of GLFW will be added to
+# `LD_LIBRARY_PATH` so that the launcher can use the correct one
+# depending on the desktop environment used.
+, waylandSupport ? false
```

Comment:
```suggestion
, withWaylandGLFW ? false
```

Reply:
What if there are other features down the line that may depend on this option?

It would make more sense to have a generic option, that affects all features in this category.

If you think that there is no possibility of anything else ever depending on that flag, we can rename it. I agree that the name you provided would make the most sense, but also was thinking about the future.

Reply:
I don't think that will be the case. And if that's ever the case, there might just be a breaking change to this package, which is totally fine.

Reply:
I think I want to call this `withAutoGLFW` instead, because `withWaylandGLFW` implies that it switches the package at build time. That's how the `with` prefix is usually used.

Reply:
`with` usually just means that something is shipped with something. Not necessarily that it replaces something.

Reply:
Done, I think we should be close to merge.